examples/using-JSON-YAML-source/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gatsby-starter-default",
```

Comment:
This `package.json` file should be updated with info specific to this example. Here's another one for comparison: https://github.com/gatsbyjs/gatsby/blob/master/examples/client-only-paths/package.json